pinot-core/src/main/java/com/linkedin/pinot/core/operator/MCombineOperator.java

```diff
@@ -83,24 +63,12 @@
     MAX_THREADS_PER_QUERY = Math.min(MAX_THREADS_PER_QUERY, (int)(numCores * .5));
   }
 
-  public MCombineOperator(List<Operator> retOperators, BrokerRequest brokerRequest) {
-    _operators = retOperators;
-    _isParallel = false;
-    _brokerRequest = brokerRequest;
-    _executorService = null;
-  }
-
-  public MCombineOperator(List<Operator> retOperators, ExecutorService executorService, long timeOutMs,
+  public MCombineOperator(List<Operator> operators, ExecutorService executorService, long timeOutMs,
       BrokerRequest brokerRequest) {
-    _operators = retOperators;
+    _operators = operators;
     _executorService = executorService;
```

Comment:
Could u please add an assert/annotation on executorService not being null, since we now assume that it will be non-null?